web/tests/server_events.test.js

```diff
@@ -51,14 +51,47 @@ const message_events = mock_esm("../src/message_events", {
 
 const server_events = zrequire("server_events");
 
+const message = {
```

Comment:
The future plan is to use a `make_message` function to generate test data as we discussed.